lib/BackendClient.js

```diff
@@ -237,29 +237,31 @@ var BackendClient = (function(){
         return deferred.promise;
     }
 
-    function registerEntryInDatabase(entryObject, state, event) {
+    function registerEntryInDatabase(entryObj, state, event) {
```

Comment:
why are you changing this method?  
1.this is not good, since we call this method every minute....
2. you should never call createRecordedEntry since it will cause race between the two DC